applepay/payment.py

```diff
@@ -62,6 +64,26 @@ def _generate_symmetric_key(self, shared_secred):
 
         return sha.digest()
 
+    @staticmethod
+    def signing_time_is_valid(signature, current_time, threshold):
```

Comment:
I don't think a default makes sense here, it's probably best to make the threshold explicit rather than implicit.